<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingAndVolumeDialog</class>
 <widget class="QDialog" name="RingAndVolumeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonRingVolumeSave">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>380</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonRingVolumeReturn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>480</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>12</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameRingAmdVolumeSetting">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>12</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>22</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>铃声</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ringvalue_1">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>85</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>铃声音量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ringvalue_2">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>155</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>通话音量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ringvalue_3">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>225</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>拨打电话</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ringvalue_4">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>295</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>拨入电话 </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ringvalue_5">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>365</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>门口主机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ringvalue_6">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>435</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>管理中心机1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ringvalue_7">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>505</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>管理中心机2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderRingVolume">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>85</y>
     <width>321</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderVolume">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>155</y>
     <width>321</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonRingVolumeDail">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>225</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>定制铃声</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonRingVolumeCallIn">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>295</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>定制铃声</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonRingVolumeDoor">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>365</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>定制铃声</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonRingVolumeCenter1">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>435</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>定制铃声</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonRingVolumeCenter2">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>505</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>定制铃声</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonRingVolumeSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
